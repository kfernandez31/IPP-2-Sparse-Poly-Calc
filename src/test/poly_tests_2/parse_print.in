-0
PRINT
-1
PRINT
2
PRINT
(-0,0)
PRINT
(1,0)
PRINT
(-4,0)
PRINT
0
PRINT
1
PRINT
2
PRINT
(0,0)
PRINT
(1,0)
PRINT
(-23,8)
PRINT
(234,13)
PRINT
((1,2)+(678,0),15)+(-7,8)
PRINT
(3,1)+(((4,4),100),2)+(56,0)
PRINT
(832,8)+(-855,4)+(634,7)+(-560,1)+(-607,2)+(583,3)+(701,9)+(-29,5)+(-106,6)+(6789,0)
PRINT
(832,8)+(-589,0)+(-855,4)+(634,7)+(-560,1)+(-607,2)+(583,3)+(701,9)+(-855,4)+(345,0)+(-29,5)+(-106,6)+(634,7)
PRINT
(1,0)+(-589,0)+(-560,1)+(-607,2)+(583,3)+(-855,4)+(-29,5)+(-106,6)+(634,7)+(832,8)+(701,9)
PRINT
