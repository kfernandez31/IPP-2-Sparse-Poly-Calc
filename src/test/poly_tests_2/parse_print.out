0
-1
2
0
1
-4
0
1
2
0
1
(-23,8)
(234,13)
(-7,8)+((678,0)+(1,2),15)
(56,0)+(3,1)+(((4,4),100),2)
(6789,0)+(-560,1)+(-607,2)+(583,3)+(-855,4)+(-29,5)+(-106,6)+(634,7)+(832,8)+(701,9)
(-244,0)+(-560,1)+(-607,2)+(583,3)+(-1710,4)+(-29,5)+(-106,6)+(1268,7)+(832,8)+(701,9)
(-588,0)+(-560,1)+(-607,2)+(583,3)+(-855,4)+(-29,5)+(-106,6)+(634,7)+(832,8)+(701,9)
